actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -93,7 +93,10 @@ def render_for_browser_request(request, wrapper)
         template = create_template(request, wrapper)
         file = "rescues/#{wrapper.rescue_template}"
 
-        if request.xhr?
+        if request.format == "text/javascript"
```

Comment:
You can do `request.format.js?`. I'm kind of disappointed, I don't see a `request.js?` 😦